Source tex file for the code review for Elieen (#122)
UniversityOfSaskatchewanCMPT371/term-project-fall2019-team-2: documents/ID4/CodeReviewForEileen.tex

\documentclass{article}

\title{Code Review}
\author{Kevin, Braunson}
\date{\today}

\begin{document}

\maketitle

\section{roles}
Moderator: Kevin \\
Reviewers: Kevin, Braunson\\
Reviewee: Elieen\\

\section{objectives}
A Review of the code written by Elieen for the functionality of TimelineComponent Snapshot Test. 

\section{scoring}
    Provide a rating from 1 - 5. \\
    1 - Work has room for improvement \\
    3 - Work was done to an acceptable standard \\
    5 - The work went above and beyond expectations\\

\begin{tabular}{|l|c|}
	\hline
	Criteria & Score\\
	\hline
    Completeness:
    (completed to the full scope of the requirements) & 5 \\
	\hline
    Concise:
    (the purpose is achieved without undue complication) & 5 \\
	\hline
    Documentation (Including author): & 5\\
    \hline
    Structure/organization:
    (does the work flow well, are components grouped logically?) & 4\\
    \hline
    Modularity: & 5\\
    \hline
    Follows code style: & 5\\
    \hline
\end{tabular}

\section{Notes}

> Documentation did really well\\

Reviewer: ```suggestion
Documentation was well done, included inline comments for lines of code that were not clear.\\
```

> Split HTML with new lines make code easier to read\\

Reviewer: ```suggestion
Splitting the HTML with new lines made the snapshots easier to read\\
```

> Sample data is realistic\\

Reviewer: ```suggestion
Sample data is realistic and extensive, covering all the columns we have in some of our sample datasets\\
```

\pagebreak

\section{List of Defects}

> @ts-ignore should be removed\\

Reviewer: ```suggestion
@ts-ignore should be removed and instead a null check done to prevent the compiler warning\\
```
